vaadin-dialog-flow-parent/vaadin-dialog-flow-integration-tests/src/main/java/com/vaadin/flow/component/dialog/tests/DialogTestPage.java

```diff
@@ -192,6 +192,8 @@ private void createResizableDraggableDialog() {
         dialog.setDraggable(true);
         dialog.setWidth("200px");
         dialog.setHeight("200px");
+        dialog.setTop("50px");
+        dialog.setLeft("50px");
```

Comment:
Looks like this breaks an existing test (`draggableDialog_shouldAllowDraggingFromDivContainer`). Is this needed?

Reply:
Removed.